### ISSUE TITLE
Wrongly handled request errors without response from Freebox

### ISSUE THREAD

**Issue by nouknouk** [matschik/freebox#48]
Hi,

in function `request`, we have:

```
} catch (error) {
    const {status, data} = error.response;
```
But in some cases, axios can return an error without a `response` field
cf. example at https://www.npmjs.com/package/axios#handling-errors
```
axios.get('/user/12345')
  .catch(function (error) {
    if (error.response) {
      // CASE 1: The request was made and the server responded with a status code
      // that falls out of the range of 2xx
    } else if (error.request) {
      // CASE 2: The request was made but no response was received
    } else {
      // CASE 3: Something happened in setting up the request that triggered an Error
    }
```
CASE 2 happened on my side (because freebox was completely down).

The Case 2 is not handled and leads to an unexpected exception: `TypeError: Cannot destructure property 'status' of 'error.response' as it is undefined.`

**Comment by matschik** [comment]
Really interesting use-cases ! Thanks a lot !!
I will fix it asap :)

**Comment by matschik** [comment]
Now available in freebox@1.3.3

**Comment by nouknouk** [comment]
Superb, thanks !
